### ISSUE TITLE
snaptron_query NA coercion error

### ISSUE THREAD

**Issue by jpling** [leekgroup/recount#5]
I'm running into an error with snaptron_query. It appears to be related to processing snaptron return values, i.e. 18 columns instead of the expected 19? Either the 'samples' or 'read_coverage_by_sample' column is missing and everything appears shifted by one. Have tried troubleshooting everything on my end, not sure what the issue is.

```
Warning messages:
1: In matrix(strsplit(jxs, "\t")[[1]], ncol = 19) :
  data length [18] is not a sub-multiple or multiple of the number of columns [19]
2: In matrix(strsplit(jxs, "\t")[[1]], ncol = 19) :
  data length [18] is not a sub-multiple or multiple of the number of columns [19]
3: In matrix(strsplit(jxs, "\t")[[1]], ncol = 19) :
  data length [18] is not a sub-multiple or multiple of the number of columns [19]
4: In (S4Vectors:::coercerToClass(element.type))(v, ...) :
  NAs introduced by coercion
5: In snaptron_query(junctions) : NAs introduced by coercion
```

**Comment by lcolladotor** [comment]
Hi @jpl135,

Snaptron recently changed (like 2 days ago) and I haven't had the time to update `recount::snaptron_query()`. I'll have this resolved by the end of next week given the end of year vacations. You can use Snaptron directly in the mean time.

Best,
Leonardo

**Comment by lcolladotor** [comment]
Hi @jpl135,

I have fixed this issue in recount 1.1.12 (Bioc-devel) and 1.0.7 (Bioc-release). Please wait 1 or 2 days before the updated packages are available via Bioconductor's biocLite() functionality.

Best,
Leonardo

**Comment by jpling** [comment]
Thank you very much Leonardo!
